ansible/roles/pgo-metrics/templates/grafana-rbac.json.j2

```diff
@@ -5,5 +5,10 @@
         "name": "grafana",
         "namespace": "{{ metrics_namespace }}"
     },
-    "automountServiceAccountToken": false
+    "automountServiceAccountToken": false,
+    "imagePullSecrets": [
+{% if ccp_image_pull_secret %}
+        { "name": "{{ ccp_image_pull_secret }}" }
```

Comment:
This works for a single element in the array, but we might want to switch to YAML if this gets any longer.